README.md

```diff
@@ -24,7 +24,7 @@ available in the image:
       * redis
       * zmq
 
-  2. PHP 7.0.26 and PHP 7.1.12
+  2. PHP 7.0.30 and PHP 7.1.18
```

Comment:
Are these extensions not installed for 7.2?

Reply:
updated